docs: Add AI Tools to QuickStarts (#47684)
supabase/supabase: apps/docs/content/_partials/quickstart_db_setup.mdx

## 1. Create a Supabase project

To start, you need a Supabase project.

Create a new Supabase project from [the Dashboard of any organization](/dashboard/new/_) you belong to.

<Admonition type="tip" title="Want to create a project programmatically?">

> You can use [the Management API](/docs/reference/api/v1-create-a-project) or ask [the MCP server](/docs/guides/ai-tools/mcp#account-management) to create a new Supabase project.

Reviewer: ```suggestion
Use [the Management API](/docs/reference/api/v1-create-a-project) or ask [the MCP server](/docs/guides/ai-tools/mcp#account-management) to create a new Supabase project.
```

Global comment (minor): Rather than telling people they can do something in response to a question, just tell them to do it directly.

> </Admonition>

Reviewer 2: Shouldn't this be a partial too?

Author: The two admonitions for the management API and MCP are actually slightly different from each other…

## 2. Set up your database

When your Supabase project is up and running, create an `instruments` table with some sample data. Then set only the privileges each Postgres role needs, add [Row Level Security (RLS)](/docs/guides/database/postgres/row-level-security) for enhanced security for database data by default, and create an RLS policy to make the data in the table publicly readable.

> Do this within your project's dashboard by copying and running the snippet in your project's [SQL Editor](/dashboard/project/_/sql/new).

Reviewer: ```suggestion
Do these steps within your project's dashboard by copying and running the snippet in your project's [SQL Editor](/dashboard/project/_/sql/new).
```

Minor: there are a few handwaving references like "this" or "them". Here, "this" seems to refer to three different things you have to do.

## 1. Create a Supabase project

> To start, you will need a Supabase project.

Author: ```suggestion
To start, you need a Supabase project.
```

> Before you can use Supabase, you need a Supabase project.

Reviewer 2: There's a lot of "Supabase" in this sentence, I would change it to:

```suggestion
To start, you will need a Supabase project.
```

Create a new Supabase project from [the Dashboard of any organization](/dashboard/new/_) you belong to.

<Admonition type="tip">

> You can also use [database.new](https://database.new) and [the Management API](/docs/reference/api/v1-create-a-project) to create a new Supabase project.

Reviewer 2: What does the Management API has to do with starting a project via the link? 🤔  It's a bit confusing since there's no immediate action related to the API.

Author: To be honest, it used to be included as an actual concrete step and I sort of wanted to make sure people realised it was a possibility, but yeah, maybe not here. Cna remove.

</Admonition>

## 2. Install MCP server (optional)

> The Supabase MCP server connects AI assistants to Supabase, allowing you to interact with your projects on your behalf. Find out more on how to add it to your client in [the MCP docs](/docs/guides/ai-tools/mcp).

Reviewer 2: ```suggestion
The Supabase MCP server connects AI assistants, allowing you to interact with your Supabase projects on your behalf. Find out more on how to add it in [the MCP docs](/docs/guides/ai-tools/mcp).
```

Reviewer 2: Client in this context can be "confusing" for some people when we are referring to the destination of all these to a "project". But maybe I am getting something wrong. In the end is a technical detail that serves no purpose to users in my opinion.

> The Supabase MCP server connects AI assistants to Supabase, allowing you to interact with your projects on your behalf. Find out more on how to add it to your client in [the MCP docs](/docs/guides/ai-tools/mcp).

Reviewer 3: The language here is contextual sounding for a procedure. For skimmability and to keep it action-oriented, I'd start the paragraph with "To X, do Y".

> To add MCP, go to [Docs link]... . For more information about the Supabase MCP, [Link]

Author: Hmm, I think in this case, I am a fan of a little context… Why should I care about adding the MCP server? However, in this case, the find out more and install links are the same, so I think I can tighten this up a bit

## 3. Set up your database

When your Supabase project is up and running, create an `instruments` table with some sample data. Then set only the privileges each Postgres role needs, add [Row Level Security (RLS)](/docs/guides/database/postgres/row-level-security) for enhanced security for database data by default, and create an RLS policy to make the data in the table publicly readable.

> You can do this visually with the dashboard or by using [the MCP server](/docs/guides/ai-tools/mcp) to ask your AI assistant to run the SQL snippet in your project.

Reviewer 2: ```suggestion
Do this visually within your project's dashboard or by using [the MCP server](/docs/guides/ai-tools/mcp) to ask your AI assistant to run the SQL snippet in your project.
```

Maybe a bit personal taste, but we use a lot the _"You can also do..."_ bit, which is irrelevant, it's better to be more imperative in my opinion. Especially when giving sequential instructions.

Author: Agreed, I had removed some of that, will do more.
